tfjs-backend-webgpu/src/max_pool2d_backprop_webgpu.ts

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
Will `MaxPool3DBackpropProgram` share the same file? If yes, maybe we should remove `2d` from file name?

Reply:
Currently, I am not sure the file will be shared by `3d`, so we keep the name with `2d` here. If we put the `3d` in this file, then we could rename the file.